internal/orchestrion/gls.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package orchestrion
+
+import _ "unsafe" // for go:linkname
+
+var (
+	// getDDGLS returns the current value from the field inserted in runtime.g by orchestrion.
+	// Or nil if orchestrion is not enabled.
+	getDDGLS = func() any { return nil }
+	// setDDGLS sets the value in the field inserted in runtime.g by orchestrion.
+	// Or does nothing if orchestrion is not enabled.
+	setDDGLS = func(any) {}
+)
+
+//go:linkname _DdOrchestrionGlsGet __dd_orchestrion_gls_get
+var _DdOrchestrionGlsGet func() any
```

Comment:
don't hesitate to use a more friendly name here ;-)

Reply:
At first it was the same name but golangci-lint was not happy about it